internal/manifests/var.go

```diff
@@ -131,10 +141,38 @@ func serviceNameQueryFrontendHTTP(stackName string) string {
 	return fmt.Sprintf("loki-query-frontend-http-%s", stackName)
 }
 
+func signingServiceSecretName(stackName string) string {
+	return fmt.Sprintf("%s-%s", stackName, "metrics")
+}
+
 func fqdn(serviceName, namespace string) string {
 	return fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, namespace)
 }
 
+func volumeMounts(podSpec corev1.PodSpec, stackName string) corev1.podSpec {
+	adjusted := podSpec
+
+	args := adjusted.Containers[0].Args
+	volumeMounts := adjusted.Containers[0].VolumeMounts
+
+	volumeMounts = append(volumeMounts, corev1.VolumeMount{
+		Name:      signingServiceSecretName(stackName),
+		MountPath: "/etc/proxy/secrets",
+	})
+	volumeMounts = append(volumeMounts,corev1.VolumeMount{
+		Name:      "certificates",
+		MountPath: "/etc/proxy/loki",
+	})
```

Comment:
:x: Why is the purpose of two volumenMounts? FWIW the cert signing service is creating only one secret which translates in one Volume registration in the pod spec and one volume mount into the Loki component container.